backend/audit/templates/audit/submission-progress.html

```diff
@@ -191,7 +203,10 @@ <h1 class="border-base-light margin-top-0">{{ auditee_name | default_if_none:''
 				<div class="usa-icon-list__content {% if additional_UEIs_workbook.created is True %} text-green {% else %} text-base-dark {% endif %}">
 					<a class="usa-link text-bold"
 						href="/report_submission/additional-ueis/{{ report_id | default_if_none:'' }}">Upload the Additional UEIs workbook</a>
-				</div>
+						<p class="margin-0">
+							This workbook is only necessary if multiple the audit report covers UEIs. It is completed by the auditee. List the additional UEIs covered by the audit, excluding the primary UEI.
```

Comment:
`only necessary if multiple the audit report covers UEIs.`
Not sure if this needs correction, but wanted to flag it.

Reply:
Fixed